packages/schematics-core/src/test-utils.ts

```diff
@@ -55,7 +55,7 @@ export async function runSchematic(opt?: {
   collectionPath?: string;
 }): Promise<UnitTestTree> {
   const schematicRunner = new SchematicTestRunner(
-    name,
+    opt?.name || 'schematics',
```

Comment:
@schuchard Not sure about this line. It showed an error.. so i fixed it like so.
Tests did work. What do you think?